app/views/hiring_staff/vacancies/_show.html.haml

```diff
@@ -0,0 +1,124 @@
+- content_for :page_title_prefix do
+  #{@vacancy.job_title} - #{@vacancy.school_name}
+
+.vacancy.grid-row
+  - if @vacancy.expired?
+    .column-full
+      .banner-warning
+        = t('jobs.expired')
+
+  .column-full
+    .share-url.display-none
+      %h2.heading-small.mt0 Share this job
+      #{link_to t('jobs.view_public_link'), @vacancy.share_url, id: 'share-url', 'aria-label': t('jobs.view_public_link')}, or #{link_to t('jobs.copy_public_url'), '#', class: 'copy-to-clipboard', 'data-clipboard-text': @vacancy.share_url, 'aria-label': t('jobs.copy_public_url')}
```

Comment:
how does a link_to work on an `aria-label`? Does it just generate the full url?

Reply:
It's just using the translations?